vicon2_driver/src/vicon2_driver.cpp

```diff
@@ -121,11 +110,11 @@ ViconDriverNode::ViconDriverNode(const rclcpp::NodeOptions node_options)
 // In charge of choose the different driver options related and provided by the Vicon SDK
 void ViconDriverNode::set_settings_vicon()
 {
-  ViconDataStreamSDK::CPP::Result::Enum result(ViconDataStreamSDK::CPP::Result::Unknown);
+  Result::Enum result(Result::Unknown);
   if (stream_mode_ == "ServerPush") {
-    result = client.SetStreamMode(ViconDataStreamSDK::CPP::StreamMode::ServerPush).Result;
+    result = client.SetStreamMode(StreamMode::ServerPush).Result;
   } else if (stream_mode_ == "ClientPull") {
-    result = client.SetStreamMode(ViconDataStreamSDK::CPP::StreamMode::ClientPull).Result;
+    result = client.SetStreamMode(StreamMode::ClientPull).Result;
   } else {
     RCLCPP_FATAL(get_logger(), "Unknown stream mode -- options are ServerPush, ClientPull");
     rclcpp::shutdown();
```

Comment:
shutdown here?

Reply:
Dunno, that was there since first commit. Probably we don't need to be so extreme.